### ISSUE TITLE
Unconsistent interpolation

### ISSUE THREAD

**Issue by arnaudon** [BlueBrain/eFEL#268]
Hello, 
in some cases, efel does not interpolate time and voltages consistently, here is a reproducing example, with data in this zip file:
[traces_bug.pkl.zip](https://github.com/BlueBrain/eFEL/files/10812641/traces_bug.pkl.zip)

```
import efel
import pickle

data = pickle.load(open('traces_bug.pkl', 'rb'))
efel.setDoubleSetting("interp_step", 0.05)
feat = efel.getFeatureValues([data], ['initburst_sahp'])
print(feat)
```

The crash is here: 
```  
File "eFEL/efel/pyfeatures/pyfeatures.py", line 165, in initburst_sahp
    sahp_interval = voltage[numpy.where(
IndexError: index 80001 is out of bounds for axis 0 with size 80001 
```
I suspect the interpolation has a small bug in this case, which appears only if I set `inter_step` to values of 0.05 and below. With 0.06, `feat` is 
```
[{'initburst_sahp': array([-86.62030036])}]
```
thanks for your help!

**Comment by wvangeit** [comment]
@AurelienJaquier Can you have a look?
Be a bit careful with changes to the interpolate function though. We don't want to change it's exact behavior, unless if it involves a bug. It can also easily lead to memory access issues, so make sure to use asserts to protect things.

**Comment by arnaudon** [comment]
Can't you use np.interp? ;P

**Comment by AurelienJaquier** [comment]
This has been fixed in MR #270, so I'll close this issue. @arnaudon don't hesitate to re-open if you still have issues with interpolation.

**Comment by arnaudon** [comment]
Lovely, thank you!
